invenio_workflows/alembic/a26f133d42a9_create_invenio_workflows_tables.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2017 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Create invenio_workflows tables."""
+
+from __future__ import absolute_import, print_function
+
+import uuid
+
+from alembic import op
+from datetime import datetime
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy_utils.types import ChoiceType, UUIDType, JSONType
+from workflow.engine_db import WorkflowStatus
+from invenio_workflows.models import ObjectStatus
+
+# revision identifiers, used by Alembic.
+revision = 'a26f133d42a9'
+down_revision = '720ddf51e24b'
+branch_labels = ()
```

Comment:
Shouldn't this depend on the dbd one from invenio_db?

Reply:
Can you confirm that the only thing that is created there is the transaction table? Do we really need this dependency?

Reply:
I don't think we really need  it, but iirc it was some kind of convention on invenio side to start from there. I'm convinced either way :)